doc/build-windows.md

```diff
@@ -6,8 +6,11 @@ Below are some notes on how to build Bitcoin Core for Windows.
 Most developers use cross-compilation from Ubuntu to build executables for
 Windows. This is also used to build the release binaries.
 
-Currently only building on Ubuntu Trusty 14.04 is supported.
-Other versions are unsupported or known to be broken (e.g. Ubuntu Xenial 16.04).
+Building on Ubuntu Trusty 14.04 is recommended.
+It is possible to build on Ubuntu Xenial 16.04. Follow the Windows Subsystem for Linux
+steps since it uses the same Ubuntu version.
+Building on Ubuntu Zesty 17.04 has been verified to work using the same steps as for
```

Comment:
I think you can write "17.04 or later", since 17.10 seems to work as well with the same steps.